pipeline/gene_to_nissl.py

```diff
@@ -0,0 +1,218 @@
+# Copyright 2021, Blue Brain Project, EPFL
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Script that maps Gene Expression to Nissl."""
+from __future__ import annotations
+
+import argparse
+import json
+import logging
+import pathlib
+import sys
+from pathlib import Path
+
+import nrrd
+import numpy as np
+from skimage.color import rgb2gray
+
+from atlannot.ants import register, transform
+
+# Initialize the logger
+logger = logging.getLogger("gene-to-nissl")
+DATA_FOLDER = pathlib.Path(__file__).resolve().parent.parent / "data"
```

Comment:
Again, I would not mind deleting this and forcing the user to provide.


I would not mind having these global variables / defaults in the final `full_pipeline`, however, IMO we should not put them in these scripts independent scripts

Reply:
Done in [0dd7092](https://github.com/BlueBrain/Deep-Atlas/pull/8/commits/0dd70926fa73cc4191788c50981dd9029b62b891).